.github/workflows/tests.yml

```diff
@@ -31,4 +32,32 @@ jobs:
 
     - name: Build with Poetry
       run: |
-        poetry build
\ No newline at end of file
+        poetry build
+
+  docs:
+    name: Build Documentation
+    runs-on: ubuntu-latest
+    needs: tests
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.10'
+
+    - name: Install Poetry
+      run: |
+        pip install poetry
+        poetry config virtualenvs.create false
+
+    - name: Install Dependencies
+      run: poetry install
+
+    - name: Install Dependencies and Build Documentation
+      run: |
+        cd docs
+        pip3 install -r requirements.docs.txt
+        mkdocs gh-deploy --force
```

Comment:
@erev0s this command doesn't just build the docs but also deploys them to https://androguard.github.io/androguard (by modifying the `gh-pages` branch), which I doubt is intended *when the tests are run on any PR*.

Reply:
fair point - was thinking that different PRs would be modifying the docstrings and it would be nice to see if something messes things -- but maybe its an overkill.